tutorials/suspend-rendering.html

```diff
@@ -0,0 +1,363 @@
+
+            #logOuput div:first-child {
+              font-weight: bold;
+            }
+        </style>
+        <script data-jsfiddle="example1">
+            var data1 = [
+              [1, 'Gary Nash', 'Speckled trousers', 'S', 1, 'yes'],
+              [2, 'Gloria Brown', '100% Stainless sweater', 'M', 2, 'no'],
+              [3, 'Ronald Carver', 'Sunny T-shirt', 'S', 1, 'no'],
+              [4, 'Samuel Watkins', 'Floppy socks', 'S', 3, 'no'],
+              [5, 'Stephanie Huddart', 'Bushy-bush cap', 'XXL', 1, 'no'],
+              [6, 'Madeline McGillivray', 'Long skirt', 'L', 1, 'no'],
+              [7, 'Jai Moor', 'Happy dress', 'XS', 1, 'no'],
+              [8, 'Ben Lower', 'Speckled trousers', 'M', 1, 'no'],
+              [9, 'Ali Tunbridge', 'Speckled trousers', 'M', 2, 'no'],
+              [10, 'Archie Galvin', 'Regular shades', 'uni', 10, 'no']
+            ];
+
+            var data2 = [
+              [11, 'Gavin Elle', 'Floppy socks', 'XS', 3, 'yes'],
+            ];
+
+            var data3 = [
+              [12, 'Gary Erre', 'Happy dress', 'M', 1, 'no'],
+              [13, 'Anna Moon', 'Unicorn shades', 'uni', 200, 'no'],
+              [14, 'Elise Eli', 'Regular shades', 'uni', 1, 'no']
+            ];
+
+            var hot = new Handsontable(document.getElementById('example1'), {
+              data: data1,
+              width: 700,
+              height: 250,
+              colHeaders: ['ID', 'Customer name', 'Product name', 'Size', 'qty', 'Return'],
+              licenseKey: 'non-commercial-and-evaluation',
+            });
+
+            function alterTable() {
+              hot.alter('insert_row', 10, 10);
+              hot.alter('insert_col', 6, 1);
+              hot.populateFromArray(10, 0, data2);
+              hot.populateFromArray(11, 0, data3);
+              hot.setCellMeta(2, 2, 'className', 'green-bg');
+              hot.setCellMeta(4, 2, 'className', 'green-bg');
+              hot.setCellMeta(5, 2, 'className', 'green-bg');
+              hot.setCellMeta(6, 2, 'className', 'green-bg');
+              hot.setCellMeta(8, 2, 'className', 'green-bg');
+              hot.setCellMeta(9, 2, 'className', 'green-bg');
+              hot.setCellMeta(10, 2, 'className', 'green-bg');
+              hot.alter('remove_col', 6, 1);
+              hot.alter('remove_row', 10, 10);
+              hot.setCellMeta(0, 5, 'className', 'red-bg');
+              hot.setCellMeta(10, 5, 'className', 'red-bg');
+            }
+
+            Handsontable.dom.addEvent(buttonWithout, 'click', function() {
+              standardAlterTable();
+            });
+
+            Handsontable.dom.addEvent(buttonWith, 'click', function() {
+              batchedAlterTable();
+            });
+
+            function standardAlterTable() {
+              const t1 = performance.now();
+
+              alterTable();
+              const t2 = performance.now();
+
+              logOutput(`Time without batch ${t2 - t1} ms`);
```

Comment:
```suggestion
              logOutput(`Time without batch ${(t2 - t1).toFixed(2)} ms`);
```
With a fixed number, it looks a little bit better. What do you think?

Reply:
As the code has to be rewritten to ES3 I make a change within https://github.com/handsontable/docs/pull/267/commits/534c5de25c9f65c4afe4b8f1f5153e2dfab40273.